src/MicrosoftTeams.ts

```diff
@@ -1531,6 +1538,11 @@ namespace microsoftTeams {
      * Apps should use this as a signal to prevent any changes to content associated with archived teams.
      */
     isTeamArchived?: boolean;
+
+    /**
+     * The type of the host client. Possible values are : android, ios, web, desktop
+     */
+    hostClientType?: string;
```

Comment:
It's probably best to use an enum for the hostClientType instead of a simple string